config/ext_jenkins-infra.yaml

```diff
@@ -218,7 +218,7 @@ controller:
                 sshKeysCredentialsId: "ec2-agents-privkey"
                 useInstanceProfileForCredentials: false
                 templates:
-                - ami: "ami-0a15ad8528897469a" # https://github.com/jenkins-infra/packer-images/ LINUXAMD64 (DO NOT DELETE OR EDIT THIS COMMENT, USED BY UPDATECLI: https://github.com/jenkins-infra/kubernetes-management/blob/cd7e4132eef4095ca287e8c17b284d3c04234cb4/updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml#L73)
+                - ami: "{{ source "getLatestUbuntuAgentAMIAmd64" }}" # https://github.com/jenkins-infra/packer-images/ LINUXAMD64 (DO NOT DELETE OR EDIT THIS COMMENT, USED BY UPDATECLI: https://github.com/jenkins-infra/kubernetes-management/blob/cd7e4132eef4095ca287e8c17b284d3c04234cb4/updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml#L73)
```

Comment:
Well something weird happened here, probably because of depends_on

Reply:
🤔 Is it the feature "depends on which would cause it, or is it the deprecated form of `depends_on`?

I'm asking because since @lemeurherve updated the manifest (https://github.com/jenkins-infra/kubernetes-management/blob/main/updatecli/updatecli.d/charts/jenkins-agents-infra.ci.jenkins.io.yaml#L27)

Reply:
The problem is the doc generated from the jsonschema suggests to use "dependson" which doesn't work.
This [PR](https://github.com/updatecli/updatecli/pull/886) officially support both `dependson` and `depends_on` and show a warning in the later case.

So either we rollblack from `dependson` to `depends_on` or I publish a new release.

I was considering pushing a release later today or tomorrow 🤷🏾‍♂️